src/stateManagement/reducer.ts

```diff
@@ -0,0 +1,28 @@
+import { ActionType, GlobalStateInterface } from "./types";
+import { initialState } from "./GlobalStore";
+
+const Reducer = (state: GlobalStateInterface, action: ActionType): any => {
+  switch (action.type) {
+    case 'MERGE_OPTION':
+      let payload = action.payload;
+      return {
+        ...state, wizardOptions: {...state.wizardOptions, ...payload}
+      };
+    case 'NEXT_STEP':
+      return {
+        ...state, wizardStep: state.wizardStep + 1
+      }
+    case 'PREV_STEP':
+      return {
+        ...state, wizardStep: state.wizardStep - 1
+      }
+    case 'PURGE_STATE':
+      return initialState;
+    default:
+      return state;
+
+  }
+
+}
+
+export default Reducer;
```

Comment:
@RobertPorterfield  -- Haven't see this before --- Github doesn't like that you don't have a newline at the end of the file.
![image](https://user-images.githubusercontent.com/97044303/169814895-0d735c0a-9f77-4cce-a170-44904b36f215.png)